hs_app_netCDF/models.py

```diff
@@ -51,18 +51,24 @@ def create(cls, **kwargs):
 
         :param kwargs: the 'value' in kwargs should be a dictionary
         """
-
+        value_arg_dict = None
         if 'value' in kwargs:
+            value_arg_dict = kwargs['value']
+        elif '_value' in kwargs:
+            value_arg_dict = json.loads(kwargs['_value'])
```

Comment:
@ironfroggy @pkdash I followed the current metadata implementation to use JSON.loads() to get a dictionary input serialized into JSON formats. See https://github.com/hydroshare/hydroshare/blob/develop/hs_core/models.py#L765-L797 for detailed docstring. @pkdash What do you think of @ironfroggy's comment here? Is there anything that needs to be changed on Coverage metadata class implementation regarding this _value and value field or how input is passed in? @ironfroggy any specific suggestions on what to do to improve or fix things here?